File: FCUBEAPP/src/app/models/billregisterrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Billregisterrptmodel } from "./billregisterrptmodel";

export class Billregisterrptlistmodel {

    billRegisterRptList: Billregisterrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: FCUBEAPI/FreightMasters/Models/BillRegisterRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Shared.Models;

namespace FreightMasters.Models
{
    public class BillRegisterRptListModel
    {
        public List<BillRegisterRptModel> BillRegisterRptList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
